### PR TITLE
[serving] update api call of get_online_service

### PR DESCRIPTION
**Pull request by aviaIguazio**
fixing bug [ML-5590](https://jira.iguazeng.com/browse/ML-5590)
updating the api call of get_online_feature_service to FeatureVector.get_online_feature_service()

### CODE REVIEW

File: mlrun/serving/routers.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 import mlrun.common.model_monitoring
 import mlrun.common.schemas.model_monitoring
 from mlrun.utils import logger, now_date
+import mlrun.feature_store as fs
```
Comment (gtopper) [inline mlrun/serving/routers.py:32]:
```suggestion
import mlrun.feature_store as fstore
```
fs can mean feature set or feature store, and for this reason we changed it everywhere to disambiguate.

File: mlrun/serving/routers.py [hunk 2]
```diff
@@ -1162,7 +1163,7 @@ def __init__(
 
     def post_init(self, mode="sync"):
         super().post_init(mode)
-        self._feature_service = mlrun.feature_store.get_online_feature_service(
+        self._feature_service = fs.FeatureVector.get_online_feature_service(
             feature_vector=self.feature_vector_uri,
```
Comment (gtopper) [inline mlrun/serving/routers.py:1167]:
```suggestion
        self._feature_service = self.feature_vector.get_online_feature_service(
```

File: mlrun/serving/routers.py [hunk 3]
```diff
@@ -1305,7 +1306,7 @@ def __init__(
...
```
Comment (gtopper) [inline mlrun/serving/routers.py:1310]:
```suggestion
        self._feature_service = self.feature_vector.get_online_feature_service(
```

File: mlrun/serving/routers.py [hunk 4]
```diff
@@ -29,6 +29,7 @@
 import mlrun.common.model_monitoring
 import mlrun.common.schemas.model_monitoring
 from mlrun.utils import logger, now_date
+import mlrun.feature_store as fstore
```
Comment (assaf758) [inline mlrun/serving/routers.py:32]: are you sure you need this?

Comment (aviaIguazio) [reply mlrun/serving/routers.py]: removed

**Review by assaf758 — APPROVED** [review]

**Pull request merged**